dali/operators/generic/reduce/reduce.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/generic/reduce/reduce.h"
+
+
+namespace dali {
+
+DALI_SCHEMA(ReduceBase)
+  .AddOptionalArg(
+    "axes",
```

Comment:
Also support "axis_names", perhaps?

Reply:
For sure. How about doing that in next PR? This is nice extension of the API, but right now this op can already be useful as it is.

Reply:
Fine with me. There are already several operators using it (SliceAttr family, Erase,...). Maybe it'd be good to make a PR that extracts that to AxesAttr or something like this, so we reuse the implementation and the arg documentation.